www-client/luakit/luakit-2017.07.26.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="A fast, light, simple to use micro-browser using WebKit and Lua."
+HOMEPAGE="https://aidanholm.github.io/luakit/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/aidanholm/luakit.git"
+else
+	SRC_URI="https://github.com/aidanholm/luakit/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="doc luajit"
+
+RDEPEND="
+	dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-lua/luafilesystem[luajit=]
+	net-libs/webkit-gtk:4=
+	x11-libs/gtk+:3
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( dev-lang/lua:0 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )"
+
+DOCS=()
```

Comment:
I'm supressing default DOCS because the Makefile installs the documentation already during the install stage:
https://github.com/aidanholm/luakit/blob/develop/Makefile#L90-L91